File: SOURCE_DOCSUITE/VecompSoftware.DocSuiteWeb.Gui/App/Models/Dossiers/DossierRoleModel.ts
import RoleModel = require('App/Models/Commons/RoleModel');
import AuthorizationRoleType = require('App/Models/Commons/AuthorizationRoleType');
import DossierModel = require('./DossierModel');

class DossierRoleModel  {
    UniqueId: string;
    AuthorizationRoleType: AuthorizationRoleType;
    Role: RoleModel;
    IsMaster: boolean;
    EntityShortId: number;
    Dossier: DossierModel;
    Status: string;
}

export = DossierRoleModel;

File: SOURCE_WEBAPI/VecompSoftware.DocSuiteWeb.Model/Entities/Dossiers/DossierRoleModel.cs

using System;
using VecompSoftware.DocSuiteWeb.Model.Commons;
using VecompSoftware.DocSuiteWeb.Model.Entities.Commons;

namespace VecompSoftware.DocSuiteWeb.Model.Entities.Dossiers
{
    public class DossierRoleModel
    {
        public Guid UniqueId { get; set; }

        public AuthorizationRoleType Type { get; set; }

        public RoleModel Role { get; set; }

    }
}
